ext/telemetry/errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(tracker) {
+  return {
+    trackUncaught: () => {
+      window.onerror = (msg, url, line, col, err) => {
```

Comment:
Ok, my mistake, I thought they were using the node event, but they actually use `window.onerror` (https://github.com/atom/atom/blob/5f82b34a0ed77af170a8b3c643a51fe246ac6a99/src/atom-environment.coffee#L749-L767). So we have to make sure we restore their callback after the install.